sdk/communication/azure-communication-callingserver/azure/communication/callingserver/utils/_utils.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse # type: ignore
+import validators
+
+from azure.core.exceptions import (
+    ClientAuthenticationError,
+    HttpResponseError,
+    ResourceExistsError,
+    ResourceNotFoundError,
+)
+
+class CallingServerUtils(object):
+
+    @staticmethod
+    def is_valid_url(
+        url # type: str
+    ): # type: (...) -> bool
+        result = validators.url(url)
+
+        if isinstance(result, validators.utils.ValidationFailure):
+            return False
+
+        return True
+
+    @staticmethod
+    def parse_length_from_content_range(content_range):
+        '''
+        Parses the content length from the content range header: bytes 1-3/65537
+        '''
+        if content_range is None:
+            return None
+
+        # First, split in space and take the second half: '1-3/65537'
+        # Next, split on slash and take the second half: '65537'
+        # Finally, convert to an int: 65537
+        return int(content_range.split(' ', 1)[1].split('/', 1)[1])
```

Comment:
If the content range is not set correctly, like it has no '/', will this throw a null ref, and if it does, is it handled up the stack?

Reply:
The content range header that this receives is a [well known standard](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Range) where the space and the slash have to exist. Please let me know that we should assume that we may be breaking the standards in our server responses.